### DISCUSSION TITLE
Question sur le mot cle await (correction quizz)

### DISCUSSION THREAD

**Discussion by Pismice** [web-classroom/web-classroom.github.io#22] [Q&A : Théorie / Cours] [answered]
Bonsoir,

Je ne comprends pas le résultat: **Les données puis After fetch**.

Pour moi comme le mot clé **await** n'est pas présent on ne va pas attendre la **Promise** en retour de la fonction et donc continuer le flow d execution sans attendre que la fonction soit resolve.

On devrait donc pour moi avoir "After fetch" -> les données.

D ailleurs en executant le code dans le navigateur j'obtiens ma version et non celle du corrigé.

Est-ce que mon raisonement est incorrect ?

![image](https://github.com/web-classroom/web-classroom.github.io/assets/33352709/521369e1-7c6c-4af6-a042-f6eba4fcb4b7)

**Comment by OlivierLmr** [comment, marked as answer]
Désolé de ne pas vous avoir répondu plus tôt, et merci de m'avoir demandé avant le début du cours. Pour la postérité, c'est effectivement une erreur et on aurait donc effectivement 'After fetch' avant les données.
